test/common/stats/allocator_impl_test.cc

```diff
@@ -434,9 +415,9 @@ TEST_F(AllocatorImplTest, AskForDeletedStat) {
 }
 
 TEST_F(AllocatorImplTest, ForEachSinkedCounter) {
-  std::unique_ptr<TestSinkPredicates> moved_sink_predicates =
-      std::make_unique<TestSinkPredicates>();
-  TestSinkPredicates* sink_predicates = moved_sink_predicates.get();
+  std::unique_ptr<Stats::TestUtil::TestSinkPredicates> moved_sink_predicates =
```

Comment:
s/Stats::// here -- we are already in `Stats`

Reply:
done

Reply:
Still doesn't look done. You need the TestUtil:: but I don't think you need the Stats::.